docs/source/operator_reference.rst

```diff
@@ -4,23 +4,49 @@
 Operators
 =========
 
-Loading operators.
-
-.. raw:: html
-
-    <iframe
-      src="_static/dpf.html#CPython"
-      style="
-        position: fixed;
-        top: 36px;
-        bottom: 0px;
-        right: 0px;
-        width: 100%;
-        border: none;
-        margin: 0;
-        padding: 0;
-        overflow: hidden;
-        z-index: 1000;
-        height: 100%;
-      ">
-    </iframe>
+DPF operators allow the manipulation and the transformation of simulation data.
+
+From DPF Server for Ansys 2023R2 onwards, the operators available in DPF will depend on the active
```

Comment:
Could we maybe add a description of what is available before 2023R2?

Reply:
@cbellot000 Do you mean a list of Operators?